src/api/api.ts

```diff
@@ -1,11 +1,13 @@
 import {Octokit} from "@octokit/rest";
 import {version} from "../../package.json";
+import {getGitHubApiUri} from "../configuration/configuration";
 
 export const userAgent = `VS Code GitHub Actions (${version})`;
 
 export function getClient(token: string): Octokit {
   return new Octokit({
     auth: token,
-    userAgent: userAgent
+    userAgent: userAgent,
+    baseUrl: `${getGitHubApiUri()}`
```

Comment:
```suggestion
    baseUrl: getGitHubApiUri()
```

Reply:
Must have left that from when I tried manually tagging the /api/v3 onto the url, implemented by e6fa56779d7445e1ccbbce5cbbc3112922c1529c